nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HistoricHealthCheckEventRetriever.java

```diff
@@ -110,11 +105,6 @@ public void execute(AdminCommandContext context) {
             actionReport.setActionExitCode(ActionReport.ExitCode.FAILURE);
             return;
         }      
-        if (!service.isHistoricalTraceEnabled()) {
-            actionReport.setMessage("Health Check Historical Trace is not enabled!");
-            actionReport.setActionExitCode(ActionReport.ExitCode.FAILURE);
```

Comment:
Correct me if I'm being dumb, but would it have worked to just change this line to a success code? Only because I like it returning a 'disabled' message when it's disabled. It's more of a question than a review really, so feel free to disregard it!